### DISCUSSION TITLE
Please allow for coloured status lables for new Projects system

### DISCUSSION THREAD

**Discussion by LukeWolfGames** [community/community#18078] [Accessibility]
Colour is important when managing certain issues and tasks.
Allowing us to customise the status with a coloured background of our choice will really help with organising in this system.

**Comment by benjiallen** [comment]
@LukeWolfGames can you help me understand your feedback more? Can you provide a screenshot of what you would like to customize?

Also, are there any [accessibility considerations](https://www.w3.org/WAI/fundamentals/accessibility-intro/) which are driving this feedback? For example, color customization might help with color blindness. If not, no worries, we'll just move this discussion to a different category.

**Reply by LukeWolfGames** [reply]
![image](https://user-images.githubusercontent.com/28833199/172742220-9625a8e8-511a-41fc-aee1-24e71501e94d.png)

The items here we should be able make coloured labels. Mainly for helping notice certain things going on but yes it could help with colour-blindness on the side.
I thought accessibility was the most relevant option when posting this thread but feel free to move this wherever.

**Comment by cryptoespanto68** [comment]
> Colour is important when managing certain issues and tasks. Allowing us to customise the status with a coloured background of our choice will really help with organising in this system.

**Comment by cehfisher** [comment]
Hello @LukeWolfGames - I wanted to follow up on this stale issue. Have the recent project theme changes resolved it for you? If so, could you please close the issue? If not, what additional changes are needed? Thanks! (cc @queenofcorgis)

**Comment by LukeWolfGames** [comment]
Hello. The colour is showing in projects but not in issues.

**Reply by cehfisher** [reply]
Thank you @LukeWolfGames for the updated context on what has been fixed and what still needs to be looked into. Could you please provide a link or screenshot of what you are seeing today, so I can check to see if there is an internal feature request yet? If not, I can create that for you and follow-up on the progress. Thanks!

**Comment by LukeWolfGames** [comment]
![image](https://github.com/community/community/assets/28833199/c24aa13d-eec9-44bf-8fd6-07f8f4d52d64)
This part when viewing an issue thread:

**Reply by cehfisher** [reply]
Thank you for the additional context @LukeWolfGames! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Reply by cehfisher** [reply]
We wanted to follow up on this reported issue. We were unable to reproduce the lack of color for status field labels even on the issue itself. Here is a screenshot of a status field label with an emoji and the color red applied to it.
<img width="320" alt="Label with an emoji of a woman detective and the text To triage in red." src="https://github.com/user-attachments/assets/a57cf483-09ad-4631-a717-708862c2ff0f">

Were your status field labels created awhile ago? It could be that you just need to update the label color to clean out the cache. Could you check under Projects -> Settings -> Status then choose an option, click the three dots to the right of the option, then Edit option. This should open up a modal which you can change the label text, color, and description. Once a new color has been applied, click Save. Now go back to an issue and hard refresh the page to see that color applied on the issue level. Hopefully this will resolve the issue for you!

**Reply by cehfisher** [reply]
Hi @LukeWolfGames - we wanted to follow-up and ask which theme you are _not_ seeing the color changes in? The screenshot you provided looks like it may be high contrast dark, but wanted to confirm before closing this issues out as not reproducible. Thanks!

**Comment by Nat-67** [comment]
> El color es importante a la hora de gestionar determinados asuntos y tareas. Permitirnos personalizar el estado con un fondo de color de nuestra elección nos ayudará mucho a organizarnos en este sistema.
